### DISCUSSION TITLE
JSON report not found error

### DISCUSSION THREAD

**Discussion by ejmarunde** [orta/vscode-jest#1031] [Q&A]
After the test run, I'm getting the following error:
`[error] JSON report not found at C:\Users\{user_name}\AppData\Local\Temp\jest_runner_{app_name}_{user_name}.json`

The actual report is being written to the root project directory with a name of:
`Users{user_name}appDataLocalTempjest_runner_{app_name}_{user_name}.json`.
* Note the file name includes the expected output directory path just smashed together.

How can I configure it to either write the file to the AppData\Local\Temp directory, or look for the file in the workspace root (with a more appropriate name?

**Comment by connectdotz** [comment]
This looks like a bug... the report path is generated, not configurable... do you have a sample repo we can see what is going on?
